test/CMakeLists.txt

```diff
@@ -8,3 +8,11 @@ add_custom_target(test-force
         --builddir=${PROJECT_BINARY_DIR}
         --vardir=${PROJECT_BINARY_DIR}/test/var
         --force)
+
+add_custom_target(test-flaky
+    COMMAND ${CMAKE_COMMAND} -E env RANGE=$$RANGE
+        git diff --relative=test --name-only $$\{RANGE:-master..HEAD\}
+        -- ./*/*test.lua
+        | paste -sd' ' - | awk '{for (i=0\; i<32\; i++) print $$0}'
```

Comment:
Why do we need this line? Wouldn't it work already work with xargs below? And why 32 times, while the commit message says 64?